examples/demo2/main.rs

```diff
@@ -30,16 +30,16 @@ mod tabs;
 mod term;
 mod theme;
 
-pub use app::*;
+pub use app::App;
 use color_eyre::Result;
-pub use colors::*;
-pub use term::*;
-pub use theme::*;
+pub use colors::{color_from_oklab, RgbSwatch};
+pub use term::{init, restore};
+pub use theme::THEME;
```

Comment:
`cargo +nightly fmt` should split these because of them being modules / external… So something seems fishy here.

Reply:
(changed this to `pub use crate::{...}` and removed the unnecessary exports)

Reply:
I think self:: would be clearer as it's an example. Yeah, an example is a crate too, but it's less obvious. Reading it from self seems clearer to me.

Reply:
done